### PR TITLE
Update .markdownlint.json

### PR DESCRIPTION
**Pull request by darksun27**
Fixed the typos and grammar

### CODE REVIEW

**Comment by ljharb** [comment]
(replaces #1906)

File: linters/.markdownlint.json [hunk 1]
```diff
@@ -117,38 +117,38 @@
...
-  "comment": "MD038: Disallow spaces inside code span elements",
+  "comment": "MD038: Disallow spaces inside code span elements.",
   "no-space-in-code": true,
 
-  "comment": "MD039: Disallow spaces inside link text",
+  "comment": "MD039: Disallow spaces inside the linked text.",
   "no-space-in-links": true,
 
-  "comment": "MD040: Fenced code blocks should have a language specified",
+  "comment": "MD040: Fenced code block should have a language specified.",
```
Comment (ljharb) [inline linters/.markdownlint.json]: `blocks` is proper english here

File: linters/.markdownlint.json [hunk 2]
```diff
@@ -18,71 +18,71 @@
...
   "header-start-left": true,
 
-  "comment": "MD024: Disallow multiple headers with the same content",
+  "comment": "MD024: Disallow multiple headers with the same content.",
   "no-duplicate-header": true,
 
-  "comment": "MD025: Disallow multiple top level headers in the same document",
-  "comment": "Gotta have a matching closing brace at the end",
+  "comment": "MD025: Disallow multiple top level headers in the same document.",
+  "comment": "You must have a matching closing brace at the end.",
```
[the remark this answers is not in the document]
Comment (darksun27) [reply linters/.markdownlint.json]: No actually it just sounded better so suggested this

File: linters/.markdownlint.json [hunk 3]
```diff
@@ -18,71 +18,71 @@
...
+  "comment": "MD012: Disallow multiple consecutive blank lines.",
   "no-multiple-blanks": {
     "maximum": 1
    },
 
   "comment": "MD013: Line length",
   "line-length": false,
 
-  "comment": "MD014: Disallow dollar signs used before commands without showing output",
+  "comment": "MD014: Disallow use of dollar signs($) before commands without showing output.",
```
[the remark this answers is not in the document]
Comment (darksun27) [reply linters/.markdownlint.json]: Should i revert it or leave it like this?

[the remark this answers is not in the document]
Comment (ljharb) [reply linters/.markdownlint.json]: Leave it :-)

**Comment by ljharb** [comment]
Please don't close and reopen PRs; if jobs need rerunning, maintainers will do that manually.

**Pull request merged**